main.py

```diff
@@ -17,7 +17,7 @@
                 bot.load_extension(full_name)
                 print(f'Loaded {full_name}')
             except (discord.ClientException, ModuleNotFoundError):
-                print(f'Failed to load extension {extension}.')
+                print(f'Failed to load extension {extension}.')  
```

Comment:
```suggestion
                print(f'Failed to load extension {extension}.')
```

It appears you've accidentally added a whitespace here.